src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -356,7 +372,16 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
 
             Uri callbackUrl = _urlResolver.ResolveRouteNameUrl(RouteNames.AadSmartOnFhirProxyCallback, new RouteValueDictionary { { "encodedRedirect", Base64UrlEncoder.Encode(redirectUri.ToString()) } });
 
-            // TODO: Deal with client secret in basic auth header
+            try
```

Comment:
Could this be combined with the try-catch on line 350?

Reply:
Yes 👍